src/main/java/org/kairosdb/datastore/cassandra/CassandraDatastore.java

```diff
@@ -630,16 +624,10 @@ public long calculateRowTimeWrite(long timestamp) {
         return (timestamp - (Math.abs(timestamp) % m_rowWidthWrite));
     }
 
-    private List<Long> calculateReadTimeBuckets(long startTime, long endTime) {
-        LinkedList<Long> buckets = new LinkedList<Long>();
+    private List<Long> calculateReadTimeBuckets(long startTimeBucket, long endTimeBucket) {
```

Comment:
I'd just return `IntStream.range(startTimeBucket, endTimeBucket)` from this method

Reply:
we have a bucket size (not just `i++`)